array_analyzer/workflows/well_wf.py

```diff
@@ -0,0 +1,79 @@
+import sys, getopt, os
+
+from array_analyzer.extract.image_parser import *
+
+import time
+from datetime import datetime
+import skimage.io as io
+import pandas as pd
+import string
+
+
+def well_analysis(input_folder_, output_folder_, debug=False):
```

Comment:
refactor filename and function name to be more obvious: `elisa_workflow` and this function `elisa_analysis`